README.md

```diff
@@ -4,5 +4,101 @@
+
+```julia
+#test images for the folder
+slcSubsetPath = "../data/s1a-iw3-slc-vv_subset_hight9800_10400_width11000_11000.tiff"; 
+slc_image = SARProcessing.load_tiff(slcSubsetPath);
+grd_image_hom = SARProcessing.load_tiff("../data/grd_image_hom.tiff"); 
+slc_image[1:4,1:4:end] 
+```
+and displayed by using the `sar2gray` function
+
+```julia
+slc_image = abs.(slc_image);
+SARProcessing.sar2gray(slc_image, p_quantile = 0.95)
+```
+
+### Speckle Reduction
+
+A common SAR processing step is speckle reduction. The objective of it is to reduce the variance of the pixels such that a better estimate of the intensity is found. The trade-off is a poorer spatial resolution. There are many speckle filters that all try to reduce the black-white granularity. The `speckle_mean_filter` function introduces the mean filter
+
+```julia
+descpek_mean_9 = SARProcessing.speckle_mean_filter(slc_image,[11,11]);
+SARProcessing.sar2gray(descpek_mean_9, p_quantile = 0.95)
+```
+
+The mean filter is a simple filter that assign the mean value of a region to a center pixel.
+
+### Object Detection
+
+Object detection is used in many fiels of Earth Observation in both the land and maritime enviroment. For the maritime enviroment, it is used in great extent for, e.g., iceberg detection or ship detection. Often, object detection is used as a priliminary analysis whereafter the detected objects are classified as belonging to different classes.
+
+We use CA-CFAR and CP-CFAR object detectors to find objects in an image. First, the size of both the guard and background (clutter) windows must be defined
+
+```julia
+background_window = 12; 
+guard_window = 7;
+probability_for_alarms = 10^(-12);
+target_window=3;
+```
+
+Using the CA-CFAR to find objects in the image
+
+```julia
+image_cp_cfar = SARProcessing.constant_false_alarm_rate_with_convolution_and_pooling(slc_image.^2,background_window,guard_window,probability_for_alarms);
+SARProcessing.sar2gray(image_cp_cfar, p_quantile = 0.9)
+```
+
+Using the CP-CFAR
+
+```julia
+image_cp_cfar = SARProcessing.constant_false_alarm_rate_with_convolution_and_pooling(slc_image.^2,background_window,guard_window,probability_for_alarms);
+SARProcessing.sar2gray(image_cp_cfar, p_quantile = 0.95)
+```
+
+Getting location of objects and subset are further described in the [tutorial] (https://github.com/AIRCentre/JuliaEO/blob/main/notebooks/hands_on_sessions/Working_with_SAR_and_InSAR_Data/SARProcessing/3_Object_detection.ipynb).
+
+### InSAR
+
+
+
+## Project Status
+
+The package is under development and the first version 0.1 will be released soon. Breaking changes are expected and some core missing features will be added. Upon that package version, 1.0 will be released and registered as an official Julia package.
+
+As with all open-source software, please try it out and report your experience. We will be happy to gather any users' feedback to improve.
+
+The package is tested on the current Julia version (1.8), and the current master on macOS.
```

Comment:
The automated tests run for 1.6 and 1.8 on Mac, windows and ubuntu (Linux)

Reply:
Yeah, just like Eigil said on the chat